### ISSUE TITLE
Accessibility bug - Text components are not focusable through keyboard and screen reader won't announce it

### ISSUE THREAD

**Issue by bharathpalaksha** [microsoft/azuredatastudio#25444]
- Azure Data Studio Version: 1.47.0
- OS Version: windows 11, mac and linux

This accessibility bug is on extension but we use Azure data studio components, text component is not getting focused through keyboard, hence screen reader isn't able to announce it

Steps to Reproduce:

https://github.com/microsoft/azuredatastudio/assets/342603/7c01c682-60a3-4b5e-80bf-81ec898c817e

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by Samhitha66** [comment]
#A11yMAS; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio;
